lib/main.js

```diff
@@ -773,6 +787,13 @@ function ReturnResult(result) {
     state.defs = GLYPH.defs;
     state.n = GLYPH.n;
     state.ID = ID;
+    state.macros = {};
+    for (var macroName in TEX.Definitions.macros) {
```

Comment:
Again, `if (TeX.Definitions.macros.hasOwnProperty(maroName)) {...}` should be used inside this loop.